miso-web/src/main/webapp/pages/editRun.jsp

```diff
@@ -358,7 +358,7 @@
         </c:forEach>
       </div>
     </c:if>
-    <div id="addRunNoteDialog" title="Create new Note"></div>
+    <div id="addRunNoteDialog" title="Add Container"></div>
```

Comment:
Hmm, good point. It will be more helpful if I instead add a new dialog for the add container. Thanks!